notebooks/402-pose-estimation-webcam/402-pose-estimation.ipynb

```diff
@@ -0,0 +1,488 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Pose estimation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates live pose estimation in OpenVINO. We use [OpenPose](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/human-pose-estimation-0001) pose estimation model from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). At the bottom you can see live results with your webcam or a video file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from IPython import display\n",
+    "from numpy.lib.stride_tricks import as_strided\n",
+    "from openvino import inference_engine as ie\n",
+    "\n",
+    "from decoder import OpenPoseDecoder"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Download model\n",
+    "\n",
+    "We use omz_downloader, which is a utility from openvino-dev package. omz_downloader automatically creates a directory structure and downloads the selected model.\n",
+    "\n",
+    "If you want to download another model, please change the model name and precision. *Note: In that case, you need different pose decoder!*"
```

Comment:
```suggestion
    "If you want to download another model, please change the model name and precision. *Note: This will require a different pose decoder*"
```